news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Kyoku: Original Song & Beatmapping Contest - Results"
+date: 2024-09-19 19:30:00 +0000
+---
+
+Kyoku has concluded, so it's time to look at the final results, and check out all the amazing work by both musicians and mappers alike.
+
+![](https://assets.ppy.sh/contests/211/banner.jpg)
+
+If you don't recall, [Kyoku](/wiki/Contests/Kyoku/2024) was a contest where teams of mappers and music producers needed to work together and collaborate on creating both a brand-new original song, *and* a beatmap to go along with it. Inspired by [BMS OF FIGHTERS](https://www.bmsoffighters.net/), teams have spent the last **3 months** crafting their own works of art, all for a chance for become the top team and win both a prestigious badge, along with highly sought-after [contest points](/wiki/Contests/Contest_points).
```

Comment:
aside from "3 months" of mapping/composing it'd be good to mention that this contest started back in april (with a link to the announcement)

Reply:
https://github.com/ppy/osu-wiki/pull/12165/commits/23d01913b31d87fb86f160830ab3fd52a73fff98